hrjobcontract/CRM/Hrjobcontract/Test/Fabricator/Contact.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class CRM_Hrjobcontract_Test_Fabricator_Contact {
+
+  private static $default = [
+    'contact_type' => 'Individual',
+    'first_name'   => 'John',
+    'last_name'    => 'Doe',
+    'sequential'   => 1
+  ];
+
+  public static function fabricate($params) {
+    $params                 = array_merge(self::$default, $params);
+    $params['display_name'] = "{$params['first_name']} {$params['last_name']}";
+
+    $result = civicrm_api3(
+      'Contact',
+      'create',
+      $params
+    );
+
+    return $result['values'][0];
```

Comment:
Also I've noticed that if you don't pass 'sequential'=1 to the HRJobContract 'create' call, then in values there will be no '0' key but Job Contract ID instead.